admin_manual/configuration_server/config_sample_php_parameters.rst

```diff
@@ -1507,18 +1507,12 @@ This enables debug logs for the windows_network_drive app.
 Overriding Existing Parameter Values Using Environment Variables
 ----------------------------------------------------------------
 
-ownCloud supports overriding existing configuration values for the web UI,
-command line, and Cron environments, using `environment variables`_.
-Doing so avoids the need to storing credentials, and other *private* data,  data
-which you never want to leak out, in either code or source repositories. 
-What’s more, by taking this approach, code changes aren’t required to deploy to
-multiple environments.
-
-To override an existing setting, you need to export an environment variable,
-which has the same name as the one which you want to override, prefixed with
-``OC_``. 
-For example, if you wanted to override the value of ``dbname``, you would set
-the environment variable ``OC_dbname``.
+ownCloud supports the ability to override the *web UI*, *command line*, and *Cron* environments’ settings by using `environment variables`_.
+By doing so, you avoid the need to store credentials and other sensitive data (such as secret keys), which you never want to store in code, and which can potentially leak out. 
```

Comment:
I feel that ending the sentence at that point will prevent the full implications of using environment variables from being communicated. Though removing the parenthesised content, I agree, won't detract or hinder the message.

On it being a convenience and not a security feature, I'm mixed on that. I agree that it's not strictly a security feature, though it is a good practice. But I don't agree that it's purely a convenience feature; though convenience it does indeed provide.

I might be just splitting hairs on this one. But it felt worth mentioning.